xbmc/video/VideoInfoTag.h

```diff
@@ -291,6 +302,7 @@ class CVideoInfoTag : public IArchivable, public ISerializable, public ISortable
 
   int m_playCount;
   CBookmark m_resumePoint;
+  static const int VALUE_NOT_SET = -10000;
```

Comment:
And use a name that indicates that it's specifically for playcount because it isn't valid for the season number (as an example). Furthermore you should use CamelCase for constants (without `_`).